docs/checks.md

```diff
@@ -440,7 +440,9 @@ The check currently looks for
 and language-specific GitHub Actions that upload the package to a corresponding
 hub, e.g., [Npm](https://www.npmjs.com/). We plan to add better support to query
 package manager hubs directly in the future, e.g., for
-[Npm](https://www.npmjs.com/), [PyPi](https://pypi.org/).
+[Npm](https://www.npmjs.com/), [PyPi](https://pypi.org/) or
+[Maven](https://maven.apache.org/) (see
+[#2687](https://github.com/ossf/scorecard/issues/2687)).
```

Comment:
Changes to this file should be done in the underlying YAML. Please see https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#updating-docs